### PR TITLE
Update docs and README.md

### PR DESCRIPTION
**Pull request by singulared**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -1,18 +1,76 @@
...
+- Issue Tracker: https://github.com/rambler-digital-solutions/aioriak/issues
+- Source Code: https://github.com/rambler-digital-solutions/aioriak
+
+Feel free to file an issue or make pull request if you find any bugs or have
+some suggestions for library improvement.
+
+License
+-------
+
+The aioredis is offered under `MIT license`_.
```
Comment (kxepal) [inline docs/index.rst]: aioredis? (:

File: aioriak/client.py [hunk 2]
```diff
@@ -338,6 +338,8 @@ def bucket(self, name, bucket_type='default'):
 
     async def get_keys(self, bucket):
         '''
+            coroutine
```
Comment (kxepal) [inline aioriak/client.py]: This `sphinxcontrib-asyncio` is used, does this note ever need? Also, it seems like incomplete.

File: docs/index.rst [hunk 3]
```diff
@@ -1,18 +1,76 @@
...
+    sphinx-quickstart on Sat Apr  2 10:53:29 2016.
+    You can adapt this file completely to your liking, but it should at least
+    contain the root `toctree` directive.
 
 Welcome to aioriak's documentation!
 ===================================
 
-Contents:
+Asyncio (:pep:`3156`) Riak client library.
+This library base on official Basho python client library
```
Comment (kxepal) [inline docs/index.rst]: "This library is based on official Basho python client one". Or even "This project is based on official Bash python client library".

File: requirements-dev.txt [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Sphinx==1.4
+sphinx-autobuild==0.6.0
+sphinx-rtd-theme==0.1.9
+sphinxcontrib-asyncio==0.1.1
```
Comment (kxepal) [inline requirements-dev.txt]: Worth to upgrade to 0.2.0 (just released) as there autodoc support was added and you use it heavily.

File: aioriak/client.py (hunk 2 above)
Comment (singulared) [inline aioriak/client.py]: As i see sphinxcontrib-asyncio does't work with sphinx-autobuild. I should copy-paste all docstring into documentation.

Comment (singulared) [inline aioriak/client.py]: Oh. now i see autodoc support in 0.2.0 thanks. Great.

File: docs/index.rst (hunk 3 above)
Comment (singulared) [inline docs/index.rst]: Thanks!

File: docs/index.rst (hunk 1 above)
Comment (singulared) [inline docs/index.rst]: Yep :+1:

File: docs/index.rst [hunk 5]
```diff
@@ -1,18 +1,76 @@
...
+Riak Search                         No
+MapReduce                           No
+Tested python versions              `3.5.0, 3.5.1 <travis_>`_
+Tested Riak versions                `2.1.3 <travis_>`_
+================================  ==============================
+
+Installation
+------------
+
+The easiest way to install aioredis is by using the package on PyPi::
```
Comment (kxepal) [inline docs/index.rst]: Also aioredis, but seems the last one.

**Pull request closed without merging**